### PR TITLE
Add Grafana Labs SECURITY.md

### PR DESCRIPTION
**Pull request by RichiH**
Signed-off-by: Richard Hartmann <richih@richih.org>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated
- [ ] Add an entry in the `CHANGELOG.md` about the changes.

### CODE REVIEW

**Review by dannykopping — APPROVED** [review]
LGTM, let's just wait for @KMiller-Grafana to review the prose.

**Review by owen-d — APPROVED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Please encrypt your message to us; please use our PGP key. The key fingerprint is:
+
+F988 7BEA 027A 049F AE8E 5CAA D125 8932 BE24 C5CA
+
+The key is available from [keyserver.ubuntu.com](https://keyserver.ubuntu.com/pks/lookup?search=0xF9887BEA027A049FAE8E5CAAD1258932BE24C5CA&fingerprint=on&op=index).
+
+Grafana Labs will send you a response indicating the next steps in handling your report. After the initial reply to your report, the security team will keep you informed of the progress towards a fix and full announcement, and may ask for additional information or guidance.
+
+**Important:** We ask you to not disclose the vulnerability before it have been fixed and announced, unless you received a response from the Grafana Labs security team that you can do so.
```
Comment (KMiller-Grafana) [inline SECURITY.md:13]:
```suggestion
**Important:** We ask that you not disclose the vulnerability before it has been fixed and announced, unless you receive a response from the Grafana Labs security team allowing you to disclose the vulnerability.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Security announcements
+
+We maintain a category on the community site called [Security Announcements](https://community.grafana.com/c/support/security-announcements),
+where we will post a summary, remediation, and mitigation details for any patch containing security fixes.
```
Comment (KMiller-Grafana) [inline SECURITY.md:18]:
```suggestion
We maintain [Security Announcements](https://community.grafana.com/c/support/security-announcements) on our community site,
where we post a summary, remediation, and mitigation details for any patch containing a security fix.
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+If you think you have found a security vulnerability, please send a report to [security@grafana.com](mailto:security@grafana.com). This address can be used for all of Grafana Labs's open source and commercial products (including but not limited to Grafana, Grafana Cloud, Grafana Enterprise, and grafana.com). We can accept only vulnerability reports at this address.
+
+Please encrypt your message to us; please use our PGP key. The key fingerprint is:
+
+F988 7BEA 027A 049F AE8E 5CAA D125 8932 BE24 C5CA
+
+The key is available from [keyserver.ubuntu.com](https://keyserver.ubuntu.com/pks/lookup?search=0xF9887BEA027A049FAE8E5CAAD1258932BE24C5CA&fingerprint=on&op=index).
+
+Grafana Labs will send you a response indicating the next steps in handling your report. After the initial reply to your report, the security team will keep you informed of the progress towards a fix and full announcement, and may ask for additional information or guidance.
```
Comment (KMiller-Grafana) [inline SECURITY.md:11]:
```suggestion
Grafana Labs will respond, indicating the next steps in handling your report. After the initial reply to your report, the security team will keep you informed of the progress towards a fix and full announcement, and may ask for additional information or guidance.
```

**Review by KMiller-Grafana — CHANGES_REQUESTED** [review]
My changes correct grammar and attempt to use present tense (as it is easier to understand).

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Reporting security issues
+
+If you think you have found a security vulnerability, please send a report to [security@grafana.com](mailto:security@grafana.com). This address can be used for all of Grafana Labs's open source and commercial products (including but not limited to Grafana, Grafana Cloud, Grafana Enterprise, and grafana.com). We can accept only vulnerability reports at this address.
```
Comment (KMiller-Grafana) [inline SECURITY.md:3]: I don't know what was meant by the parenthetical "including but not limited to ... grafana.com" part of this paragraph.  "grafana.com" is not a commercial or open source product, so it does not make sense to me.  Perhaps "grafana.com" should be changed to something like "other products listed at grafana.com".

I'm including this suggestion such that you can see grammar corrections that should be changed.

```suggestion
If you think you have found a security vulnerability, please send a report to [security@grafana.com](mailto:security@grafana.com). This address can be used for all of Grafana Labs' open source and commercial products (including, but not limited to Grafana, Grafana Cloud, Grafana Enterprise, and grafana.com). We accept only vulnerability reports at this address.
```

**Review by KMiller-Grafana — CHANGES_REQUESTED** [review]

**Pull request closed without merging**